index.html

```diff
@@ -202,9 +202,10 @@ <h3>Sampling and Reporting Rate</h3>
       interval between samples, expressed in milliseconds.
```

Comment:
```suggestion
      interval between samples to be obtained from the hardware,
      expressed in milliseconds.
```

This bit was removed from the PressureObserverOptions section but not added here.

Reply:
Yeah, I was unsure how valuable that info was given it is not even clear we can do it at that frequency

Reply:
but I am fine with adding it!